kvm-install-vm

```diff
@@ -337,27 +349,35 @@ function fetch_images ()
         fedora29)
           QCOW=Fedora-Cloud-Base-29-1.2.x86_64.qcow2
           OS_TYPE="linux"
-          OS_VARIANT="fedora29"
+          OS_VARIANT=$(get_fedora_os_variant fedora29)
```

Comment:
Instead of hardcoding `fedora29` here, couldn't you reuse `$DISTRO` for this and the other OS_VARIANT subshells below? In fact, I think you could opt not to pass `$1` here, and just reference `$DISTRO` inside `get_fedora_os_variant()`. Thoughts?